scripts/us_census/pep/US_PEP_SEXRACE /preprocess.py

```diff
@@ -0,0 +1,148 @@
+from County.county_1990_2000 import process_county_1990_2000
+from County.county_2000_2009 import process_county_2000_2009
+from County.county_2010_2020 import process_county_2010_2020
+from postprocess import _create_single_csv, _generate_mcf, _generate_tmcf
+
+FLAGS = flags.FLAGS
+default_input_path = os.path.dirname(
+    os.path.abspath(__file__)) + os.sep + "config_files"
+flags.DEFINE_string("input_path", default_input_path, "Import Data File's List")
+
+
+def _get_urls(json_file_path, key):
+    '''
+    Extracting dataset urls from
+    json_config_files
+    '''
+    _URLS_JSON = None
+    with open(json_file_path, encoding="UTF-8") as file:
+        _URLS_JSON = json.load(file)
+    urls = _URLS_JSON[key]
+    return urls
+
+
+def process(config_files, sv_list=None):
+    '''
+    this method calls the required
+    methods and generate final csv, mcf and tmcf
+    '''
+    flag = None
+
+    for config_file in config_files:
+        files = _get_urls(config_file, "urls")
+        if "national_1980_1990.json" in config_file:
+            process_national_1980_1990(files)
+        elif "national_1900_1970.json" in config_file:
+            process_national_1900_1970(files)
+        elif "national_1990_2000.json" in config_file:
+            process_national_1990_2000(files)
+        elif "national_2000_2010.json" in config_file:
+            process_national_2000_2010(files)
+        elif "national_2010_2020.json" in config_file:
+            process_national_2010_2020(files)
+        elif "state_1970_1979.json" in config_file:
+            process_state_1970_1979(files)
+        elif "state_1980_1990.json" in config_file:
+            process_state_1980_1990(files)
+        elif "state_1990_2000.json" in config_file:
+            process_state_1990_2000(files)
+        elif "state_2000_2010.json" in config_file:
+            process_state_2000_2010(files)
+        elif "state_2010_2020.json" in config_file:
+            process_state_2010_2020(files)
+        elif "county_1970_1979.json" in config_file:
+            process_county_1970_1979(files)
+        elif "county_1980_1989.json" in config_file:
+            process_county_1980_1989(files)
+        elif "county_1990_2000.json" in config_file:
+            process_county_1990_2000(files)
+        elif "county_2000_2009.json" in config_file:
+            process_county_2000_2009(files)
+        elif "county_2010_2020.json" in config_file:
+            process_county_2010_2020(files)
+
+    if sv_list is None:
+        # list of output files which are processed as is
+        CSVLIST = [
+            "nationals_result_1900_1959.csv", "nationals_result_1960_1979.csv",
+            "nationals_result_2000_2010.csv", "state_result_2000_2010.csv",
+            "state_result_2010_2020.csv", "county_result_2000_2009.csv",
+            "county_result_2010_2020.csv"
+        ]
+
+        # list of output files which are having aggregation Count_Person_Male
+        # and Count_Person_Female
```

Comment:
list of output files which are having aggregation 
E.g., Count_Person_Male, Count_Person_Female

Reply:
Added E.g., to the comment